autoload/neomake/makers/ft/typescript.vim

```diff
@@ -7,8 +7,9 @@ endfunction
 function! neomake#makers#ft#typescript#tsc()
     return {
         \ 'args': [
-            \ '-m', 'commonjs', '--noEmit'
+            \ '--project', getcwd(), '--noEmit'
         \ ],
+        \ 'append_file': 0,
```

Comment:
I don't know `tsc`, but this looks like it should be a project maker, i.e. in `autoload/neomake/makers/typescript.vim`?!
To be run with `Neomake!` then.

Please not that there is also `'cwd': '%:h'`, which could be used to change to the file's dir for the maker execution.

Also, please consider creating a PR / discussing this in Neomake's repo.

I found this when looking into https://github.com/neomake/neomake/issues/720.